core-v1/src/main/scala/algebra/Expr.scala

```diff
@@ -256,6 +256,7 @@ object Expr {
     override def withDebugging(debugging: Debugging[Any, Any]): Const[O, OP] = copy(debugging = debugging)
   }
 
+  // TODO: Rename to CombineWith or BinaryOperation to fit better with Zip2With?
   /**
     * Zips the output of the [[leftExpr]] and [[rightExpr]] (executed in parallel, if supported) and passes
     * the tuple as arguments to the [[operation]] that produces a single output value.
```

Comment:
`Expr.Combine` is a bit vague. I got it from `Monoid.combine(a, b)`, but it isn't really even the same signature. I should probably do this before v1.0.0, but it might be a separate PR to avoid too much churn.